### DISCUSSION TITLE
有條件的自駕車!

### DISCUSSION THREAD

**Discussion by Timmy110403540** [1101-GS3073-C/Python-Community-GS3073-C#145] [General]
德國近期允許賓士的一項技術，DRIVE PILOT（賓士輔助駕駛軟體），讓駕駛可以在限制情況下將雙手放開，讓車子自己行駛。
而限制是:「必須行駛於高速公路上，且時速低於 60 公里」。而因為德國高速公路沒有速限，所以若在60以下就可以說是塞車了。因此，自駕技術只能使用在塞車的情況下。當脫離塞車後，車子又回交回控制權給主人，若主人沒有接手，則會自動鎖車。
我認為這是一項非常棒的技術，能在塞車時做自己的事，不必受塞車所苦。而脫離塞車的機制也非常好，不再自駕，並提醒主人可以上路了!
![image](https://user-images.githubusercontent.com/90882519/146376909-000c50ff-703f-4800-bc71-bcc4e85d0544.png)

文章來源:https://technews.tw/2021/12/10/benz-level3-autopilot-approval/

**Comment by pocper1** [comment]
如果在台灣應用，想必可解決過年時，國道停車場之苦!

**Comment by bonny123865** [comment]
而且車禍的發生也對駕駛人友善很多，讓使用自駕期間的責任歸咎於開發公司，不會讓駕駛在自己不清楚電腦控制的情況下承擔後果。

**Comment by briankuo123** [comment]
給台灣那些喜歡在高速公路塞車時亂踩煞車的人用效果一定很好

**Reply by pocper1** [reply]
應該不會有駕駛會一直亂踩煞車吧...

**Comment by Jay-722** [comment]
好爽喔不用自己開車，可以吃個漢堡之類的

**Comment by wh20502** [comment]
我記得自駕系統好像有分5還6級，這應該是第三級，我覺得相關法規或是出事了的責任歸屬，可能比新技術來得更加重要。

**Comment by jack90325** [comment]
塞車的時候可以直接開始玩手游了，謝謝賓士

**Comment by yasklop** [comment]
好方便的發明，幫助大家省了好多時間，這樣塞車時就能做其他事了

**Comment by handsomeniko** [comment]
其實自駕車有分6級，另外，自駕車車禍的責任歸屬也是很熱門的議題。

**Comment by Shiba010** [comment]
好像特斯拉早就有做出差不多的輔助系統了，

**Comment by 110409538** [comment]
我認為這是一項很不錯的技術，可以利用塞車的時間做其他事情。

**Comment by ghost** [comment]
雙手放開坐在駕駛座上 車子往前開 感覺好奇怪www
